jsx/MultiSelectDropdown.js

```diff
@@ -287,24 +297,38 @@ class SelectDropdown extends Component {
         selectLabel = this.props.selectedCategory;
       }
     }
+    const overlay = this.state.open ? (
+       <div style={{
+         top: 0,
+         left: 0,
+         zIndex: 100,
+         position: 'fixed',
+         width: 'calc(100vw)',
+         height: 'calc(100vh)',
+        }} onClick={this.overlayClickHandler}
+       />
+     ) : null;
     return (
-      <div className={parentDivClass}>
-        <button type="button"
-                className="btn btn-default dropdown-toggle col-xs-12"
-                onClick={this.toggleDropdown}>
-          <div className="col-xs-10">
-              <span className="pull-left">
-                {selectLabel}
-              </span>
-          </div>
-          <div className="pull-right">
-            <span className="glyphicon glyphicon-menu-down"></span>
-          </div>
-        </button>
-        <ul className="dropdown-menu">
-          {options}
-        </ul>
-      </div>
+      <>
```

Comment:
I don't see any reason to use a fragment here. It should just be contained in a real element.

Reply:
@driusan the {overlay} on line 330 is a separate div than the div on line 313 and so the fragment is needed.